widgets/contact/contact.php

```diff
@@ -682,14 +683,8 @@ function sanitize_multiple_emails( $value ) {
 	}
 
 	function modify_instance( $instance ) {
-		// Use this to set up an initial version of the
-		if ( empty( $instance['settings']['to'] ) ) {
-			$current_user               = wp_get_current_user();
-			$instance['settings']['to'] = $current_user->user_email;
```

Comment:
@braamgenis I removed this (it now uses `default_domain_email` instead) as it's unlikely this email won't match up with the site admin email and it will prevent running an additional check for something that will likely be changed anyway. Should I revert this change?